### PR TITLE
Added comments as requested.

### PR DESCRIPTION
**Pull request by Malind19**

### CODE REVIEW

**Comment by illfated** [comment]
Ref. issue ticket #3353

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/configuration/customize-and-export-start-layout.md [hunk 1]
```diff
@@ -155,6 +155,8 @@ When you have the Start layout that you want your users to see, use the [Export-
 >* If you place executable files or scripts in the \ProgramData\Microsoft\Windows\Start Menu\Programs folder, they will not pin to Start.
 >
 >* Start on Windows 10 does not support subfolders. We only support one folder. For example, \ProgramData\Microsoft\Windows\Start Menu\Programs\Folder. If you go any deeper than one folder, Start will compress the contents of all the subfolder to the top level.
+>
+>* 3 additional shortcuts are pinned to the start menu after the export; which are of locations %ALLUSERSPROFILE%\Microsoft\Windows\Start Menu\Programs, %APPDATA%\Microsoft\Windows\Start Menu\Programs and %APPDATA%\Microsoft\Windows\Start Menu\Programs\System Tools\
```
Comment (JohanFreelancer9) [inline windows/configuration/customize-and-export-start-layout.md]:
```suggestion
>* Three additional shortcuts are pinned to the start menu after the export. These are shortcuts to %ALLUSERSPROFILE%\Microsoft\Windows\Start Menu\Programs, %APPDATA%\Microsoft\Windows\Start Menu\Programs, and %APPDATA%\Microsoft\Windows\Start Menu\Programs\System Tools\.
```

**Comment by mypil** [comment]
@Malind19 - please merge the commit on the copy/edit review made by JohanFreelancer9 / nenonix  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by Malind19** [comment]
@mypil Done

**Comment by mypil** [comment]
@DaniHalfin @Justinha @AndreaBarr @mjcaparas @jaimeo @greg-lindsay @lizap - Can anyone of you please assist to check, approve and merge the content of this PR based on issue #3353?

Thank you.

**Pull request merged**